client/blocks/app-banner/index.jsx

```diff
@@ -149,7 +157,47 @@ export class AppBanner extends Component {
 
 		const { title, copy } = getAppBannerData( translate, currentSection );
 
-		return (
+		const jetpackAppBanner = (
```

Comment:
I might create two separate components (or functions) -- one for Jetpack, and one for WordPress. That way, we don't need to execute this code even when we aren't using it.

Reply:
I like that idea, and have gone ahead to implement it in https://github.com/Automattic/wp-calypso/pull/67013/commits/e385f512e946afcf17c16034ebdcdfb4e690f0ac. It's worth noting that the code for the WordPress banner will eventually be deleted, so it may be worth revisiting this in the future when only the Jetpack banner exists.